### ISSUE TITLE
Bug: a11y - Screen reader does not read title in calcite-modal

### ISSUE THREAD

**Issue by rslibed** [Esri/calcite-components#1023]
### Summary
An accessibiility report states that the modal title of `calcite-modal` should be read when opened but the screen reader is unable to do so due to the dialog box itself not being focusable. 

### Actual Behavior
Screen reader does not read the modal title when opened. 

### Expected Behavior
Screen reader should read the modal title when opened. 

Related issue:
https://devtopia.esri.com/WebGIS/arcgis-template-configuration/issues/588

**Comment by macandcheese** [comment]
Still valid

**Comment by paulcpederson** [comment]
closing in favor of duplicate https://github.com/Esri/calcite-components/issues/1756 which is more comprehensive.
